src/components/Toggle/index.tsx

```diff
@@ -0,0 +1,210 @@
+import styled from '@emotion/styled'
+import {
+  ChangeEvent,
+  ChangeEventHandler,
+  ReactNode,
+  useCallback,
+  useState,
+} from 'react'
+import Tooltip from '../Tooltip'
+
+const SIZES = {
+  large: {
+    ball: 16,
+    height: 24,
+    width: 48,
+  },
+  small: {
+    ball: 12,
+    height: 20,
+    width: 40,
+  },
+} as const
+
+const StyledToggle = styled.div<{
+  'data-checked': boolean
+  'data-disabled': boolean
+  'data-size': 'small' | 'large'
```

Comment:
Dont need to use `data-size` because the css doesn't depend of its value. Use `size` prop and don't forget to use shouldForwardProp 😄